docs/useful-robot-commands.md

```diff
@@ -248,3 +248,20 @@ Alternative (without entering redis repl):
 
 `redis-cli get <redis_key>` or `redis-cli set <redis_key> <value>`
 
+
+## GDB Server
+
+Sometime it would be nice to gdb into the robot to see the backtrace from a segfault. Run the following commands in order to launch gdb
+
+
+Prerequisite: a `gdb-server` must be installed locally on the robot! You must be in the `src` directory when running this!
```

Comment:
you need to specify that `src` directory refers to your local machine and NOT the robot